alphapept/utils.py

```diff
@@ -315,3 +321,27 @@ def wrapper(*args, **kwargs):
         logging.debug("FUNCTION `{}` FINISHED".format(given_function.__name__))
         return result
     return wrapper
+
+def check_github_version() -> str:
+    """Checks and returns the current version of AlphaPept.
+    Parameters
+    ----------
+
+    Returns
+    -------
+    : str
+        The version on the AlphaPept GitHub master branch.
+        None if no version can be found on GitHub
+    """
+    import urllib.request
+    import urllib.error
+    try:
+        with urllib.request.urlopen(LATEST_GITHUB_INIT_FILE) as version_file:
```

Comment:
Add timeout for urllib checking, just in case of no internet connections for some servers.